### PR TITLE
doc: comment "add only reachable addresses to addrman"

### PR DESCRIPTION
**Pull request by kristapsk**
Proposed by Sjors during review of #25678, was likely just missed, as it also for me looks a code where comment will not hurt.

https://github.com/bitcoin/bitcoin/pull/25678#discussion_r964482832

### CODE REVIEW

File: src/net.cpp [hunk 1]
```diff
@@ -1648,6 +1648,9 @@ void CConnman::ThreadOpenConnections(const std::vector<std::string> connect)
 
             if (add_fixed_seeds_now) {
                 std::vector<CAddress> seed_addrs{ConvertSeeds(Params().FixedSeeds())};
+                // We will not make outgoing connection to peers that are unreachable
```
Comment (mzumsande) [inline src/net.cpp:1651]: nit: connection**s**

**Comment by mzumsande** [comment]
ACK 9094951cd09c62aa6fd2214f90e3eb87ca2d98af

File: src/net.cpp (hunk 1 above)
Comment (kristapsk) [reply src/net.cpp]: Thanks, fixed!

File: src/net.cpp [hunk 2]
```diff
@@ -1648,6 +1648,9 @@ void CConnman::ThreadOpenConnections(const std::vector<std::string> connect)
 
             if (add_fixed_seeds_now) {
                 std::vector<CAddress> seed_addrs{ConvertSeeds(Params().FixedSeeds())};
+                // We will not make outgoing connections to peers that are unreachable
+                // (e.g. because of -onlynet configuration).
+                // Therefore, it makes no sense to add them to addrman in the first place.
```
Comment (Zero-1729) [inline src/net.cpp:1653]: Slight rewording nit

```suggestion
                // Therefore, we do not add them to addrman in the first place.
```

**Review by Zero-1729 — COMMENTED** [review]
crACK a7b4f97b016eb7728d6381765269d228a4093492

**Review by vasild — APPROVED** [review]
ACK a7b4f97b016eb7728d6381765269d228a4093492

File: src/net.cpp (hunk 2 above)
Comment (Sjors) [reply src/net.cpp:1653]: or: `, so don't add them to addrman.`

**Review by Sjors — COMMENTED** [review]
We should also mention the caveat in #26035, unless that's fixed.

**Comment by kristapsk** [comment]
> We should also mention the caveat in #26035, unless that's fixed.

I would hope "until it's fixed". :)

**Comment by kristapsk** [comment]
Addressed additional review comments.

**Review by amovfx — COMMENTED** [review]
I'm confused about the broken checks, all I see is a doc string.

File: src/net.cpp [hunk 3]
```diff
@@ -1648,6 +1648,14 @@ void CConnman::ThreadOpenConnections(const std::vector<std::string> connect)
 
             if (add_fixed_seeds_now) {
                 std::vector<CAddress> seed_addrs{ConvertSeeds(Params().FixedSeeds())};
+                // We will not make outgoing connections to peers that are unreachable
```
Comment (Sjors) [inline src/net.cpp:1651]: Maybe it wants:
```cpp
/* Blah
 * blah
 * blah */
```

**Review by vasild — APPROVED** [review]
ACK ce4257026622287c8c981fb932681730a3c6387f

File: src/net.cpp (hunk 3 above)
Comment (kristapsk) [reply src/net.cpp:1651]: No, there are two random funcional tests failing in each failed CI run for some reason (different one in each case), should not be related with this change.

**Review by Zero-1729 — COMMENTED** [review]
re-ACK ce4257026622287c8c981fb932681730a3c6387f

File: src/net.cpp (hunk 3 above)
Comment (mzumsande) [reply src/net.cpp:1651]: The second one is new to me, I opened #26051 - but yes, obviously unrelated.

**Comment by mzumsande** [comment]
ACK ce4257026622287c8c981fb932681730a3c6387f

**Pull request merged**